root/components/Table.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+import {useTable} from 'react-table';
+
+import loopParity from '../utility/loopParity';
+
+const Table = (({columns, data}) => {
+  const {
+    getTableProps,
+    getTableBodyProps,
+    headerGroups,
+    rows,
+    prepareRow,
+  } = useTable({
+    columns,
+    data,
+  });
+
+  /* eslint-disable react/jsx-key */
```

Comment:
Why don't we want keys?

Reply:
Because `key` is expanded from `getFooProps`.